lab-katherine/__test__/lib/mocks.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const User = require('../../model/user')
+const Gallery = require('../../model/gallery')
+const faker = require('faker')
+
+const mocks = module.exports = {}
+mocks.user = {}
+mocks.gallery = {}
+
+mocks.user.createOne = function() {
```

Comment:
When running your tests I get some errors in this file saying result is not defined. I would recommend sticking with either 'result' or 'this.result' but not combining the two. The two things we want attached to the result are the user and the gallery (or just the user depending on the mock).